contactsummary/js/src/contact-summary/controllers/keyDetails.js

```diff
@@ -42,11 +42,38 @@ define([
     // Private Members //
     /////////////////////
 
+    function formatDuration(duration) {
+        var texts = [],
+            days = duration.days(),
+            months = duration.months(),
+            years = duration.years();
+
+        if (years > 0) {
+            texts.push(years + " years");
```

Comment:
You mean returning: {years: 12, months: 34, days: 56} from here, and then using ng-if in the view?